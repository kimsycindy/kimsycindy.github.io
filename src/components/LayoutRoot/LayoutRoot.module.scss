@import '../../styles/variables.scss';

.LayoutRoot {
  display: flex;
  background-color: $black;
  flex-direction: column;
  min-height: 100vh;
}
