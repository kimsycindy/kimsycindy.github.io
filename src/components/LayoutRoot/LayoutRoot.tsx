import React from 'react'
import cx from 'classnames'

import styles from './LayoutRoot.module.scss'

interface Props {
  className?: string
}

const LayoutRoot: React.FC<Props> = ({ children, className }) => (
  <div className={cx(styles.LayoutRoot, className)}>{children}</div>
)

export default LayoutRoot
