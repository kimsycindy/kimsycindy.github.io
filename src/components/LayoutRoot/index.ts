import LayoutRoot from './LayoutRoot'

export default LayoutRoot
