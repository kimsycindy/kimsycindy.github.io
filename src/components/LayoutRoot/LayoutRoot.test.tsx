import React from 'react'
import renderer from 'react-test-renderer'

import LayoutRoot from './LayoutRoot'

describe('LayoutRoot', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <LayoutRoot className="layoutRoot-mock">
          <div>LayoutRoot</div>
        </LayoutRoot>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
