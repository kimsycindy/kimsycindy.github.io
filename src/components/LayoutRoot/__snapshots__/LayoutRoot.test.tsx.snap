// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutRoot renders correctly 1`] = `
<div
  className="LayoutRoot layoutRoot-mock"
>
  <div>
    LayoutRoot
  </div>
</div>
`;
